test/AngularSpec.js

```diff
@@ -240,6 +240,20 @@ describe('angular', function() {
       }
     });
 
+    it("should handle Uint8Array subarray", function() {
```

Comment:
In this test, use some type that doesn't have `BYTES_PER_ELEMENT === 1` (as does `Uint8Array`), so that we verify that everything works correctly. E.g. use `Uint16Array`.